app/ui-react/syndesis/src/modules/data/shared/VirtualizationUtils.ts

```diff
@@ -269,7 +269,7 @@ export function generateDvConnections(
       // loading (true/false)
       schemaLoading = String(virtSrcStatus.loading);
       // selection
-      if (conn.name === selectedConn) {
+      if (selectedConn && conn.name === selectedConn) {
```

Comment:
Hey Mark, I could see in a few cases it's false just because of space in the name, I was wondering is this intentional. like "SadMongo" and "Sad Mongo".

Reply:
Indra, you are correct.  I looked into this and the 'sourceName' in our returned 'status' does not match the connection names when there are spaces in it.  I have logged a separate issue to investigate (TEIIDTOOLS-912).  Good catch!